.vuepress/config.js

```diff
@@ -369,7 +369,7 @@ module.exports = {
         apiKey:
           '06UKvqod16fff6018934c85a4d393534b1b96cd6c3a5ee492bcd4a4e720e26fb24ef1cbb',
         indexUid: 'docs',
-        placeholder: 'Type / to start searching',
+        placeholder: 'Type \'s\' or \'/\' to search',
```

Comment:
Wouldn't "Press" make more sense?

Reply:
Since both options are only one letter and type usually refers to words, I guess it _would_ make more sense to use "press" or "push". In any case, I'm happy to give "Press" a try.

```suggestion
        placeholder: 'Press \'s\' or \'/\' to search',
```